### DISCUSSION TITLE
Export to hammer

### DISCUSSION THREAD

**Discussion by CN1o2** [BEEmod/BEE2-items#4186] [Tech Support / Q&A] [answered]
How can I export maps to hammer without problems?

**Comment by FrozenDragon0** [comment, marked as answer]
You would use the preview.vmf that is made with the map along with styled ones being in styled/preview.vmf

~Also please do not ask questions like this here, this is not the forum for it. Ask these questions on the discord.~

**Comment by HeimlichLaboratories** [comment]
I also do not recommend to do this, as BEE2 doesn't work too well with Hammer
